app/controllers/holidays_controller.rb

```diff
@@ -74,11 +65,51 @@ def destroy
         add_leave(@holiday.length)
       end
       @holiday.destroy
-      flash[:success] = 'Holiday was successfully destroyed.'
+      flash[:success] = t('holiday.destroyed')
       redirect_to holidays_path
     end
   end
 
+  def file
+    if ((@holiday.status.include? 'saved') || (@holiday.status.include? 'declined')) && @holiday.user = current_user
```

Comment:
Wenn du das ändern möchtest, das include? wurde im gleichen Kontext noch an anderen Stellen in diesem PR gebraucht :wink: